includes/apple-exporter/builders/class-components.php

```diff
@@ -293,6 +302,67 @@ private function anchor_together( $component, $target_component ) {
 	}
 
 	/**
+	 * Add a thumbnail if needed.
+	 *
+	 * @param array &$components
+	 * @access private
+	 */
+	private function add_thumbnail_if_needed( &$components ) {
+		// If a thumbnail is already defined, just return.
+		if ( $this->content_cover() ) {
+			return;
+		}
+
+		// Otherwise, iterate over the components and look for the first image.
+		foreach ( $components as $i => $component ) {
+			if ( is_a( $component, 'Apple_Exporter\Components\Image' ) ) {
```

Comment:
Instead of wrapping the whole thing in this conditional, I think it would be more readable if you started with
```
if ( ! is_a( $component, 'Apple_Exporter\Components\Image' ) ) {
    continue;
}
````